src/Stratis.Bitcoin.Features.Miner/PowBlockDefinition.cs

```diff
@@ -77,7 +74,6 @@ public void TestBlockValidity()
         {
             this.logger.LogTrace("()");
 
-            throw new Exception("FIX ME");
```

Comment:
Why removing this? 
It was left here so we don't forget about it while fixing the tests

Reply:
@noescape00 I removed this as it causes the mining integration tests to fail. Can we create an issue for this? Or does this link to #1934 ?

Reply:
Yes, issue #1934 is for that. 

>I removed this as it causes the mining integration tests to fail. 

Ok